### ISSUE TITLE
Tag (and APK) for latest version is missing

### ISSUE THREAD

**Issue by IzzySoft** [eprst/KTodo#4]
According to Playstore, there's a v2.2 – but the releases page has only v2.1. Could you please fill the gap? Thanks!

**Comment by eprst** [comment]
Hi

I think there's a bunch of web sites that allow to download specific apk
versions, for example:
https://apkpure.com/ktodo/com.kos.ktodo

**Comment by IzzySoft** [comment]
@eprst that might well be. But instead of sending users hunting, wouldn't it be favorable to provide it first-hand? You did so before :wink:

Apart from that, the updater for my repo (where [your app is available](https://apt.izzysoft.de/fdroid/index/apk/com.kos.ktodo) as well using the F-Droid client) cannot deal with "a bunch of web sites that allow to download specific apk", just with Github and GitLab :innocent:

**Comment by eprst** [comment]
Hi

OK, I'm sorry that I didn't provide the apk when it was just released.
Pulled 2.2 apk from the play store and created a release on github. Looks
good now?

**Comment by IzzySoft** [comment]
Thanks, yes! Update should appear in my repo with the next sync tomorrow.

PS: Looks like your app would also fit the inclusion criteria of the [official F-Droid repo](https://f-droid.org/). If you would like to see it there, let me know and I gladly assist you (disclosure: I'm one of the maintainers there).

**Comment by eprst** [comment]
Great

Honestly I don't care that much, but we can put it in F-Droid if that's not
too much effort on my side (sorry, quite busy with other stuff these days).
Also, who will be responsible for pushing updates there?

thanks

**Comment by IzzySoft** [comment]
As long as you tag each new release (and increase versionName/versionCode accordingly), and don't add any "proprietary stuff" or non-whitelisted Maven repos, our team will take care to build updates and publish them (disclosure: I'm one of the F-Droid maintainers). Still, should complications arise, we might need your help to solve them.

Does that sound good for you? Then I'd go ahead and "trigger the process".

**Comment by eprst** [comment]
Fine by me.
But why do you care about mvn repos? Your updates will be based on released
.apk's as I understand it, or are you actually building from source?

**Comment by IzzySoft** [comment]
Apps in *my repo* are based on the APKs, true. Apps in the "official repo" are solely relying on the source code (no binaries accepted), and are built from that. As for "external sources" like MVN repos: only some whitelisted Maven repos are permitted (there's trust involved; I'm not a dev, so I cannot explain that in detail – as far as I understood, some ship the signed source along and some don't, so the former can be "trusted more" while with the latter you cannot tell what source they've built from).

So in short: For my repo, the APK is sufficient. To go for the official repo, the APK is irrelevant but the source is important.

**Comment by IzzySoft** [comment]
@eprst another point that needs clarification for the official repo to accept your app (and would be good to add regardless of that): What license is this project using?

**Comment by IzzySoft** [comment]
@eprst a year later: any decision on the license yet?

**Comment by IzzySoft** [comment]
Errm… And again, tags (as well as APKs) are missing. Last version on Play was 2.3 before you unlisted. So any chance here?

**Comment by IzzySoft** [comment]
With the latest commit being more than 2 years ago it seems you've abandoned this app – its disappearance from PlayStore seems to confirm this. Further, the last release here is v2.2 – while the last one I've seen on Play was v2.4.3, which suggests it went close-source in between.

As hope dies last, I make one more try to get an answer – or two now: Is the app still maintained, if so under what license ([back in 2012](https://github.com/eprst/KTodo/issues/2#issuecomment-8247188) you've said WTFPL, but it's still not pointed out in the Readme or anywhere else), and where to find the updates? Oops, that were 3 questions, but I hope you don't mind…
